egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -17,7 +17,10 @@ def AddInputLayer(config_lines, feat_dim, splice_indexes=[0], ivector_dim=0):
     output_dim += len(splice_indexes) * feat_dim
     if ivector_dim > 0:
         components.append('input-node name=ivector dim=' + str(ivector_dim))
-        list.append('ReplaceIndex(ivector, t, 0)')
+        if ivector_interval == 0:
```

Comment:
I guess it would be better to make ivector_interval 10 the default, as it is required for chain models, LSTMS and BLSTMs.